hub/management/commands/import_socio_economic_status_data.py

```diff
@@ -0,0 +1,128 @@
+        "table": "areadata",
+        "exclude_countries": ["Scotland", "Northern Ireland"],
+        "comparators": DataSet.numerical_comparators(),
+        "default_value": 10,
+    }
+
+    label_names = {
+        "Higher managerial, administrative and professional occupations": "Higher professional & administrative occupations",
+        "Lower managerial, administrative and professional occupations": "Lower professional & administrative occupations",
+        "Intermediate occupations": "Intermediate occupations",
+        "Small employers and own account workers": "Small employers & self-employed",
+        "Lower supervisory and technical occupations": "Lower supervisory & technical occupations",
+        "Semi-routine occupations": "Semi-routine occupations",
+        "Routine occupations": "Routine occupations",
+        "Never worked / long-term unemployed": "Never worked / long-term unemployed",
+        "Full-time students": "Full-time students",
+    }
+    machine_readable_names = {
+        "Higher managerial, administrative and professional occupations": "higher_professional_administrative_occupations",
+        "Lower managerial, administrative and professional occupations": "lower_professional_administrative_occupations",
+        "Intermediate occupations": "intermediate_occupations",
+        "Small employers and own account workers": "small_employers_self_employed",
+        "Lower supervisory and technical occupations": "lower_supervisory_technical_occupations",
+        "Semi-routine occupations": "semi_routine_occupations",
+        "Routine occupations": "routine_occupations",
+        "Never worked / long-term unemployed": "never_worked_long_term_unemployed",
+        "Full-time students": "full_time_students",
+    }
+    machine_readable_backwards = {
+        value: key for key, value in machine_readable_names.items()
+    }
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "-q", "--quiet", action="store_true", help="Silence progress bars."
+        )
+
+    def handle(self, quiet=False, *args, **options):
+        self._quiet = quiet
+        df = self.get_dataframe()
+        self.data_types = self.create_data_types(df)
+        self.delete_data()
+        self.import_data(df)
+
+    def create_data_types(self, df):
+        if not self._quiet:
+            self.stdout.write("Creating dataset + types")
+        data_set, created = DataSet.objects.update_or_create(
+            name="consituency_socio_economic_status", defaults=self.defaults
+        )
+        data_types = []
+        for col in tqdm(df.columns, disable=self._quiet):
+            if col != "gss":
+                data_type, created = DataType.objects.update_or_create(
+                    data_set=data_set,
+                    name=f"se_{self.machine_readable_names[col]}",
+                    defaults={
+                        "data_type": "percent",
+                        "label": self.label_names[col],
```

Comment:
could maybe do `self.label_names.get(col, col)` here to avoid having to include the ones that aren't being overridden? and then maybe rename it `label_names_override`.